<response cycle="001" cvus="000" thread="001" suite="Sanity" name="test_sanity" step="001" number="001" type="get" result="Successful" url="/qualifier_Oct09_schedule.html" code="200" description="Get url" time="1384112874.78" duration="0.00744795799255" />
<testResult cycle="001" cvus="000" thread="001" suite="Sanity" name="test_sanity"  time="1384112874.78" result="Error" steps="1" duration="0.0124831199646" connection_duration="0.00744795799255" requests="1" pages="1" xmlrpc="0" redirects="0" images="0" links="0" traceback="Traceback (most recent call last):&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/FunkLoadTestCase.py&quot;, line 946, in __call__&#10;    testMethod()&#10;   File &quot;/home/mayank/temp-ankush/sanity/test_Sanity.py&quot;, line 23, in test_sanity&#10;    self.get(self.server_url, description='Get url')&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/FunkLoadTestCase.py&quot;, line 391, in get&#10;    method=&quot;get&quot;, load_auto_links=load_auto_links)&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/FunkLoadTestCase.py&quot;, line 334, in _browse&#10;    self._browser.pageImages(url, page, self)&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/PatchWebunit.py&quot;, line 176, in WTC_pageImages&#10;    sucker.feed(page)&#10;   File &quot;/usr/lib/python2.7/sgmllib.py&quot;, line 104, in feed&#10;    self.goahead(0)&#10;   File &quot;/usr/lib/python2.7/sgmllib.py&quot;, line 138, in goahead&#10;    k = self.parse_starttag(i)&#10;   File &quot;/usr/lib/python2.7/sgmllib.py&quot;, line 296, in parse_starttag&#10;    self.finish_starttag(tag, attrs)&#10;   File &quot;/usr/lib/python2.7/sgmllib.py&quot;, line 341, in finish_starttag&#10;    self.handle_starttag(tag, method, attrs)&#10;   File &quot;/usr/lib/pymodules/python2.7/webunit/IMGSucker.py&quot;, line 46, in handle_starttag&#10;    method(attributes)&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/PatchWebunit.py&quot;, line 150, in do_link&#10;    self.session.css[url] = self.session.fetch(url)&#10;   File &quot;/usr/lib/python2.7/dist-packages/funkload/PatchWebunit.py&quot;, line 360, in WF_fetch&#10;    h.endheaders()&#10;   File &quot;/usr/lib/python2.7/httplib.py&quot;, line 969, in endheaders&#10;    self._send_output(message_body)&#10;   File &quot;/usr/lib/python2.7/httplib.py&quot;, line 829, in _send_output&#10;    self.send(msg)&#10;   File &quot;/usr/lib/python2.7/httplib.py&quot;, line 791, in send&#10;    self.connect()&#10;   File &quot;/usr/lib/python2.7/httplib.py&quot;, line 772, in connect&#10;    self.timeout, self.source_address)&#10;   File &quot;/usr/lib/python2.7/socket.py&quot;, line 571, in create_connection&#10;    raise err&#10; error: [Errno 111] Connection refused&#10;" />
